### ISSUE TITLE
Poco F3 broken screen

### ISSUE THREAD

**Issue by KaaBoum** [Genymobile/scrcpy#6623]
Scrcpy works on OTG mode but doesn't find any adb devices on normal mode.

Both screen and touch screen are broken;  I can't turn on talkback with volume up + volume down and the combo tab, enter, tab, tab, enter doesn't work (even when using space instead of enter)

Any advices ?

**Comment by danielscos** [comment]
did you find any solutions?

**Comment by KaaBoum** [comment]
No, I didn't. I think I will try to open it and fix the screen without backup my data first...
